src/components/clusterProgress/ClusterProgress.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  PageSectionVariants,
+  TextContent,
+  Text,
+  Progress,
+  ButtonVariant,
+} from '@patternfly/react-core';
+import { Cluster } from '../../api/types';
+import PageSection from '../ui/PageSection';
+import HostsTable from '../clusterConfiguration/HostsTable';
+import ClusterToolbar from '../clusters/ClusterToolbar';
+import { ToolbarButton } from '../ui/Toolbar';
+
+type ClusterProgressProps = {
+  cluster: Cluster;
+};
+
+const ClusterProgress: React.FC<ClusterProgressProps> = ({ cluster }) => {
+  return (
+    <>
+      <PageSection variant={PageSectionVariants.light}>
+        <Breadcrumb>
+          <BreadcrumbItem>
+            <Link to="/clusters">Clusters</Link>
+          </BreadcrumbItem>
+          <BreadcrumbItem isActive>{cluster.name}</BreadcrumbItem>
+        </Breadcrumb>
+      </PageSection>
+      <PageSection variant={PageSectionVariants.light} isMain>
+        <TextContent>
+          <Text component="h1">{cluster.name}</Text>
+          <dl>
+            <dt>Creation started at</dt>
+            <dd>{new Date(Date.parse(cluster.installStartedAt || '')).toLocaleString()}</dd>
```

Comment:
Can we move this to a helper function?
In addition, can we use constructor for parsing the input string?

Reply:
Done.